src/test/java/uk/gov/dhsc/htbhf/page/SelectAddressPage.java

```diff
@@ -40,11 +41,15 @@ public WebElement getPostcodeLookupNotWorkingElement () {
     }
 
     public List<WebElement> getAddressOptions () {
```

Comment:
Could we rename this method to getSelectableAddressOptions

Reply:
done